src/test/java/com/scottlogic/competitionEngine/application/user/UserServiceImplTests.java

```diff
@@ -0,0 +1,32 @@
+package com.scottlogic.competitionEngine.application.user;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+
+import static org.hamcrest.Matchers.endsWith;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import org.springframework.test.web.servlet.MockMvc;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class UserServiceImplTests {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    public void DefaultMessageReturned() throws Exception{
+        this.mockMvc.perform(get("/users/authorise")).andDo(print()).andExpect(status().isOk())
```

Comment:
Just curious, what does the print() do here? Is that what gives us the "Hello world" back?

Reply:
Also a @FrancisD-SL question :)

Reply:
Nevermind - took it out and the test runs the same way, I don't think we're getting much out of having it here